R/organize_data.R

```diff
@@ -66,11 +66,12 @@ organize_data.tune_results <-
     }
     preds$.hover <- format_hover(var, ...)
     app_type <- get_app_type(original_data[[y_name]])
-    new_shiny_data(preds, y_name, app_type, num_col_names, fac_col_names, x)
+    best_config <- tune::select_best(x, tune::.get_tune_metric_names(x)[1])$.config
+    new_shiny_data(preds, y_name, app_type, num_col_names, fac_col_names, x, best_config)
   }
 # ------------------------------------------------------------------------------
 
-new_shiny_data <- function(predictions, y_name, subclass, numeric_cols, factor_cols, x) {
+new_shiny_data <- function(predictions, y_name, subclass, numeric_cols, factor_cols, x, best_config) {
```

Comment:
Did you add any tests for this new return value in the constructor (as well as in the `testthat` code)?